en/docs/learn/export-user-info-to-mailChimp.md

```diff
@@ -0,0 +1,30 @@
+#Exporting WSO2 Identity Server User Information to MailChimp
+
+
+MailChimp is an all‑in‑one Marketing Platform. 
+This topic provides instructions on exporting user data from WSO2 Identity Server (WSO2 IS) to the 
+MailChimp. 
+
+
+Let's get started!
+
+!!! tip "Before you begin!"
+    Create **users.csv** file by following the instructions in 
+    [Exporting User Data From WSO2 Identity Server](../learn/exporting-user-data-from-wso2-is.md).
+    
+##Try it Out
+
+1. Log in to your [MailChimp](https://mailchimp.com/) account and go to **Audience** section.
+2. Under **Add Contacts** click on **Import Contacts**.
+![](../assets/img/tutorials/mailChimp-user-provisioning/step-2.png)
+3. Select **CSV or tab-delimited test file**
+![](../assets/img/tutorials/mailChimp-user-provisioning/step-3.png)
+4. Browse the exported `users.csv` where all the users in WSO2 Identity Server are listed in and 
+proceed with the wizard.
+![](../assets/img/tutorials/mailChimp-user-provisioning/step-4.png)
+5. From the next window, map each attribute in the .csv to corresponding fields in the Mailchimp user attributes. 
```

Comment:
Map each attribute in the .csv file to corresponding fields that match the MailChimp user attributes.